src/shm_sub.c

```diff
@@ -4805,6 +4805,8 @@ sr_shmsub_listen_thread(void *arg)
              * another event is generated, our event pipe will not get notified */
             continue;
         } else if (ret) {
+            sr_errinfo_new(&err_info, ret, "sr_subscription_process_events failed with %d", ret);
```

Comment:
Why not use `sr_strerror()` on `ret`? And the message could be similar to other errors, which do not mention exact functions. Such as
```
"Processing subscription events failed (%s).", sr_strerror(ret)
```